src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -674,6 +674,124 @@ public void overdueFineShouldBeChargedWhenItemIsOverdue() {
       createdFeeFineAction.getString("typeAction"), is("Overdue fine"));
   }
 
+  @Test
+  public void overdueFineIsChargedForCorrectOwnerWhenMultipleOwnersExist() {
+    useFallbackPolicies(loanPoliciesFixture.canCirculateRolling().getId(),
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.activeNotice().getId(),
+      overdueFinePoliciesFixture.facultyStandardDoNotCountClosed().getId(),
+      lostItemFeePoliciesFixture.facultyStandard().getId());
+
+    final IndividualResource james = usersFixture.james();
+    final UUID checkInServicePointId = servicePointsFixture.cd1().getId();
+    final IndividualResource homeLocation = locationsFixture.basedUponExampleLocation(
+      item -> item.withPrimaryServicePoint(checkInServicePointId));
+    final IndividualResource nod = itemsFixture.basedUponNod(item ->
+      item.withPermanentLocation(homeLocation.getId()));
+
+    final IndividualResource checkedOutLoan = loansFixture.checkOutByBarcode(nod, james,
+      new DateTime(2020, 1, 1, 12, 0, 0, DateTimeZone.UTC));
+
+    for (int i = 0; i < 10; i++) {
+      feeFineOwnersClient.create(new FeeFineOwnerBuilder()
+        .withId(UUID.randomUUID())
+        .withOwner("fee-fine-owner-" + i)
+      );
+    }
```

Comment:
If we just added this to `overdueFineShouldBeChargedWhenItemIsOverdue()` would it test both things at the same time?